aws-lc-rs/build.rs

```diff
@@ -26,4 +28,17 @@ fn main() {
         );
         std::process::exit(1);
     }
+
+    // For PQ KEM KATs we need to enable a private aws-lc API.
+    // If AWS_LC_RUST_PRIVATE_INTERNALS=1 is set and bingen is enabled then enable our
+    // configuration to allow the tests to be built and enabled.
+    // Additionally these APIs are not thread safe. So force tests to run single threaded.
+    if env::var("AWS_LC_RUST_PRIVATE_INTERNALS")
+        .unwrap_or("0".into())
+        .eq("1")
+        && cfg!(feature = "bindgen")
+    {
+        println!("cargo:rustc-cfg=private_api");
+        println!("cargo:rustc-env=RUST_TEST_THREADS=1");
+    }
```

Comment:
Here and in the sys-crates, we should also rebuild on change to the environment variable:
```
println!("cargo:rerun-if-env-changed=AWS_LC_RUST_PRIVATE_INTERNALS");
```

Reply:
We discussed offline removing these KATs since they are covered by AWS-LC, and we agreed that the round-trip tests in aws-lc-rs should thus be sufficient. I have removed the sys crate changes and the corresponding private_api crate feature.